App.tsx

```diff
@@ -150,6 +150,7 @@ const App = () => {
       Lato_700Bold_Italic,
       Lato_900Black,
       Lato_900Black_Italic,
+      NAC_Icons: require('./assets/fonts/nac-icons.ttf'),
```

Comment:
with the font, I think you don't need `avalanche-icon.png` in this PR anymore

Reply:
Oops, good catch!